src/test/java/kitchenpos/application/ProductServiceTest.java

```diff
@@ -0,0 +1,245 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import fixture.MenuFixture;
+import fixture.ProductFixture;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuRepository;
+import kitchenpos.domain.Product;
+import kitchenpos.domain.ProductRepository;
+import kitchenpos.infra.PurgomalumClient;
+
+@ExtendWith(MockitoExtension.class)
+class ProductServiceTest {
+	@Mock
+	private ProductRepository productRepository;
+
+	@Mock
+	private MenuRepository menuRepository;
+
+	@Mock
+	private PurgomalumClient purgomalumClient;
+
+	@InjectMocks
+	private ProductService productService;
+
+	private Product validProduct;
+
+	@BeforeEach
+	void setUp() {
+		validProduct = ProductFixture.createValidProduct();
+	}
+
+	@Nested
+	class create {
+		@Test
+		@DisplayName("상품 생성 시 가격이 null이면 상품 생성을 할 수 없다")
+		void createProductWithNullPrice() {
+			// given
+			validProduct.setPrice(null);
```

Comment:
`fixture`는 한번 생성한 뒤, 고정되는 값을 의미하게되는데 대다수의 테스트 코드가 동일하게 `유효한 Product` 객체를 생성한 뒤, 일부 값을 변경하여 테스트를 실행하고 있네요. 🤔 

진짜로 여러 테스트 케이스에서 공통적으로 쓰이는 값이라면 공유해서 사용할 수 있겠지만 이런식으로 매번 테스트에 필요한 값으로 변경이 필요하다면 값을 공유할 필요없이 매번 그 테스트 케이스에 맞는 객체를 생성하는게 좋을 것 같아요.  

각 테스트 케이스에 맞는 fixture를 따로 만들어서 사용해보면 어떨까요? 
(Product 뿐만 아니라 다른 테스트 케이스에도 동일하게 적용 부탁드려요. 🙏) 

```suggestion
			Product nullPriceProduct = ProductFixture.create("test상품명", null);
```